src/Cassandra.IntegrationTests/Core/EmptyColumnTests.cs

```diff
@@ -0,0 +1,275 @@
+                            column_name = "ascii",
+                            clustering_order = "ascii",
+                            column_name_bytes = "blob",
+                            kind = "ascii",
+                            position = "int",
+                            type = "ascii"
+                        },
+                        ignore_on_prepare = false
+                    }
+                });
+
+                simulacronCluster.Prime(new
+                {
+                    when = new
+                    {
+                        query = "SELECT \"\", \" \" FROM testks.testtable"
+                    },
+                    then = new
+                    {
+                        result = "success",
+                        delay_in_ms = 0,
+                        rows = new[]
+                        {
+                                new Dictionary<string, string>
+                                {
+                                    {"", "testval"},
+                                    {" ", "testval2"}
+                                }
+                            },
+                        column_types = new Dictionary<string, string>
+                            {
+                                {"", "ascii"},
+                                {" ", "ascii"}
+                            },
+                        ignore_on_prepare = false
+                    }
+                });
+
+                var mapConfig = new MappingConfiguration();
+                mapConfig.Define(
+                    new Map<TestTable>()
+                        .KeyspaceName("testks")
+                        .TableName("testtable")
+                        .PartitionKey(u => u.TestColumn)
+                        .Column(u => u.TestColumn, cm => cm.WithName(""))
+                        .Column(u => u.TestColumn2, cm => cm.WithName(" ")));
+
+                using (var cluster = EmptyColumnTests.BuildCluster(simulacronCluster))
+                {
+                    try
+                    {
+
+                        var session = cluster.Connect();
+
+                        var testTables = new Table<TestTable>(session);
+                        var test = (from t in testTables.Execute() select t).FirstOrDefault();
+                        Assert.AreEqual("testval", test.TestColumn);
+                        Assert.AreEqual("testval2", test.TestColumn2);
+
+                        var mapper = new Mapper(session, mapConfig);
+                        test = mapper.Fetch<TestTable>().FirstOrDefault();
+                        Assert.AreEqual("testval", test.TestColumn);
+                        Assert.AreEqual("testval2", test.TestColumn2);
+
+                        var tableMetadata = session.Cluster.Metadata.GetTable("testks", "testtable");
+                        Assert.IsNotNull(tableMetadata);
+
+                        var rs = session.Execute("SELECT \"\", \" \" FROM testks.testtable");
```

Comment:
Can we include a prepare statement execution?

Reply:
good idea, added